frontend/javascripts/libs/datasource.schema.json

```diff
@@ -68,7 +68,21 @@
           "properties": {
             "name": { "type": "string" },
             "category": { "enum": ["color", "segmentation"] },
-            "elementClass": { "enum": ["uint8", "uint16", "uint24", "uint32", "uint64", "float,", "double", "int8", "int16", "int32", "int64"] }
+            "elementClass": {
```

Comment:
my ide beautifies code again :thinking:
As this is accepted by prettier I suggest to keep it this way.

Reply:
Hmm, in the long run, this can be quite cumbersome. This is especially prone to merge conflicts. Can you please disable any IDE auto-formatting which differs from prettiers behavior?